blocks/header/header.css

```diff
@@ -613,6 +616,13 @@ header a.button:hover span.button-text {
     margin: 0 12px;
   }
 
+  .mega-menu div h2 > a {
```

Comment:
It might be beneficial to target the button not with h2 but with :first-child. That would give them more flexibility in changing the page